apps/web/pages/settings/organizations/[id]/onboard-members.tsx

```diff
@@ -1,64 +1,17 @@
-"use client";
-
-import { redirect, useRouter } from "next/navigation";
-
-import AddNewTeamMembers from "@calcom/features/ee/teams/components/AddNewTeamMembers";
-import { useCompatSearchParams } from "@calcom/lib/hooks/useCompatSearchParams";
-import { useLocale } from "@calcom/lib/hooks/useLocale";
-import { Meta, WizardLayout, WizardLayoutAppDir } from "@calcom/ui";
+import { getServerSideProps } from "@calcom/features/ee/organizations/pages/organization";
 
 import PageWrapper from "@components/PageWrapper";
 
-export { getServerSideProps } from "@calcom/features/ee/organizations/pages/organization";
-
-const OnboardTeamMembersPage = () => {
-  const { t } = useLocale();
-
-  return (
-    <>
-      <Meta
-        title={t("invite_organization_admins")}
-        description={t("invite_organization_admins_description")}
-      />
-      <AddNewTeamMembers isOrg={true} />
-    </>
-  );
-};
-
-OnboardTeamMembersPage.getLayout = (page: React.ReactElement) => {
-  // eslint-disable-next-line react-hooks/rules-of-hooks
-  const router = useRouter();
-  // eslint-disable-next-line react-hooks/rules-of-hooks
-  const query = useCompatSearchParams();
-
-  return (
-    <WizardLayout
-      currentStep={4}
-      maxSteps={5}
-      isOptionalCallback={() => {
-        router.push(`/settings/organizations/${query.get("id")}/add-teams`);
-      }}>
-      {page}
-    </WizardLayout>
-  );
-};
+import OnboardTeamMembersPage, { LayoutWrapper } from "~/settings/organizations/[id]/onboard-members-view";
 
-export const buildWrappedOnboardTeamMembersPage = (
```

Comment:
Did not add `buildWrappedOnboardTeamMembersPage` to `module/settings/organizations/[id]/onboard-members-view.tsx` because it's unnecessary